tests/flows/staking/poa_to_pos.go

```diff
@@ -6,8 +6,8 @@ import (
 
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
-	nativetokenstakingmanager "github.com/ava-labs/teleporter/abi-bindings/go/staking/NativeTokenStakingManager"
```

Comment:
I think similar to the contracts, this test folder should be renamed to `validatormanager`, same with the pkg name